Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://github.com/rust-bpf/rust-bcc"
 edition = '2018'
 
 [dependencies]
+anyhow = "1.0.26"
 bcc-sys = "0.12.0"
 byteorder = "1.3.1"
 failure = "0.1.5"
```

Comment:
I believe we'd want to remove `failure` here

Reply:
Oh nice! Let me switch it up then